### PR TITLE
Adapter css variables

### PR DESCRIPTION
**Pull request by Kalmaegi**
For https://github.com/microsoft/vscode/issues/165169

- [x] src/vs/workbench/contrib/markers/browser/markersViewActions.ts
- [x] src/vs/workbench/contrib/testing/browser/testingOutputPeek.ts
- [x] src/vs/editor/contrib/linkedEditing/browser/linkedEditing.ts
- [x] src/vs/workbench/contrib/interactive/browser/interactiveEditor.ts

### CODE REVIEW

File: src/vs/platform/theme/common/colorRegistry.ts [hunk 1]
```diff
@@ -363,6 +363,11 @@ export const editorFindMatchBorder = registerColor('editor.findMatchBorder', { l
 export const editorFindMatchHighlightBorder = registerColor('editor.findMatchHighlightBorder', { light: null, dark: null, hcDark: activeContrastBorder, hcLight: activeContrastBorder }, nls.localize('findMatchHighlightBorder', "Border color of the other search matches."));
 export const editorFindRangeHighlightBorder = registerColor('editor.findRangeHighlightBorder', { dark: null, light: null, hcDark: transparent(activeContrastBorder, 0.4), hcLight: transparent(activeContrastBorder, 0.4) }, nls.localize('findRangeHighlightBorder', "Border color of the range limiting the search. The color must not be opaque so as not to hide underlying decorations."), true);
 
+/**
+ * Editor linked editing color.
+ */
+export const editorLinkedEditingBackground = registerColor('editor.linkedEditingBackground', { dark: Color.fromHex('#f00').transparent(0.3), light: Color.fromHex('#f00').transparent(0.3), hcDark: Color.fromHex('#f00').transparent(0.3), hcLight: Color.white }, nls.localize('editorLinkedEditingBackground', 'Background color when the editor auto renames on type.'));
```
Comment (aeschli) [inline src/vs/platform/theme/common/colorRegistry.ts:369]: Better to leave it in src/vs/editor/contrib/linkedEditing/browser/linkedEditing.ts.
No need to change the location.

File: src/vs/editor/contrib/linkedEditing/browser/linkedEditing.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/*---------------------------------------------------------------------------------------------
```
Comment (aeschli) [inline src/vs/editor/contrib/linkedEditing/browser/linkedEditing.css:1]: As this is a new css file, don't forget to add the css import in src/vs/editor/contrib/linkedEditing/browser/linkedEditing.ts

`import 'vs/css!./linkedEditing';

File: src/vs/workbench/contrib/interactive/browser/interactiveEditor.css [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+.interactive-editor .input-cell-container:focus-within .input-editor-container .monaco-editor {
+	outline: solid 1px var(--vscode-notebook-focusedCellBorder);
+}
+
+.interactive-editor .input-cell-container .input-editor-container .monaco-editor {
+	outline: solid 1px var(--vscode-notebook-inactiveFocusedCellBorder);
+}
+
+.interactive-editor .input-cell-container .input-focus-indicator {
+	top: 8 px;
```
Comment (aeschli) [inline src/vs/workbench/contrib/interactive/browser/interactiveEditor.css:15]: remove the space before px

**Review by aeschli — CHANGES_REQUESTED** [review]

File: src/vs/platform/theme/common/colorRegistry.ts (hunk 1 above)
Comment (weartist) [reply src/vs/platform/theme/common/colorRegistry.ts:369]: ok, I thought we preferred to put them together

File: src/vs/workbench/contrib/interactive/browser/interactiveEditor.css (hunk 3 above)
Comment (weartist) [reply src/vs/workbench/contrib/interactive/browser/interactiveEditor.css:15]: get

**Review by aeschli — CHANGES_REQUESTED** [review]

File: src/vs/workbench/contrib/testing/browser/testingOutputPeek.css [hunk 4]
```diff
@@ -0,0 +1,22 @@
+/*---------------------------------------------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (weartist) [reply src/vs/workbench/contrib/testing/browser/testingOutputPeek.css:1]: yes, I forgot that

**Review by aeschli — DISMISSED** [review]

**Review by dbaeumer — DISMISSED** [review]

**Comment by aeschli** [comment]
Thanks a lot, @weartist !

**Comment by weartist** [comment]
> Thanks a lot, @weartist !

I need to double-check my own modifications, because of network reasons, I can't build at this time😂

**Review by dbaeumer — APPROVED** [review]

**Pull request merged**